frontend/src/pages/projects/screens/spawner/imageSelector/ImageVersionSelector.tsx

```diff
@@ -56,34 +68,59 @@ const ImageVersionSelector: React.FC<ImageVersionSelectorProps> = ({
     );
   });
 
+  const isSelectedImageVersionOutOfDate =
+    selectedImageVersion && isElyraVersionOutOfDate(selectedImageVersion);
+
   return (
     <FormGroup isRequired label="Version selection" fieldId="workbench-image-version-selection">
-      <Select
-        id="workbench-image-version-selection"
-        onToggle={(e, open) => setVersionSelectionOpen(open)}
-        onSelect={(e, selection) => {
-          // We know selection here is ImageVersionSelectOptionObjectType
-          if (isImageVersionSelectOptionObject(selection)) {
-            setSelectedImageVersion(selection.imageVersion);
-            setVersionSelectionOpen(false);
-          }
-        }}
-        aria-label="Image version select"
-        isOpen={versionSelectionOpen}
-        selections={selectOptionObjects.find(
-          (optionObject) => optionObject.imageVersion.name === selectedImageVersion?.name,
-        )}
-        placeholderText="Select one"
-      >
-        {options}
-      </Select>
-      <FormHelperText>
-        <HelperText>
-          <HelperTextItem>
-            Hover an option to learn more information about the packages included.
-          </HelperTextItem>
-        </HelperText>
-      </FormHelperText>
+      <Stack hasGutter>
+        <StackItem>
+          <Select
+            id="workbench-image-version-selection"
+            toggle={(toggleRef) => (
+              <MenuToggle
+                isFullWidth
+                ref={toggleRef}
+                onClick={() => setVersionSelectionOpen(!versionSelectionOpen)}
+                isExpanded={versionSelectionOpen}
+              >
+                {selectedImageVersion?.name ?? 'Select one'}
+              </MenuToggle>
+            )}
+            onSelect={(e, selection) => {
+              // We know selection here is ImageVersionSelectOptionObjectType
+              if (isImageVersionSelectOptionObject(selection)) {
+                setSelectedImageVersion(selection.imageVersion);
+                setVersionSelectionOpen(false);
+              }
+            }}
+            aria-label="Image version select"
+            isOpen={versionSelectionOpen}
+            selected={selectedImageVersion}
+            onOpenChange={(isOpen) => setVersionSelectionOpen(isOpen)}
+            shouldFocusToggleOnSelect
+          >
+            <SelectList>{options}</SelectList>
+          </Select>
+        </StackItem>
+        <StackItem>
+          {isSelectedImageVersionOutOfDate && (
+            <Alert
+              variant="info"
+              isInline
+              isPlain
+              title="A new image version is available. Select the latest image version to use Elyra for pipelines."
+            />
+          )}
+          <FormHelperText>
```

Comment:
nit: Typically FormHelperText is situated directly beneath the input. Is it meant to show even when the alert is not shown? If so, I would expect it be rendered outside of a StackItem so that there isn't extra spacing between the input and helper text.

Reply:
The mocks show the form helper text bellow the alert. I guess I could put it all inside the form helper text, but that would just give marginal spacing changes

Reply:
Up to you, just pointing out how form helper text is typically used.

Reply:
<img width="654" alt="Screenshot 2024-04-05 at 10 26 46 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/12587674/cb086f8a-ad67-4d70-969a-7a2a99fa5a1c">